ai/Algorithm/Graph/Node.py

```diff
@@ -45,6 +45,13 @@ def add_vertex(self, p_vertex):
     def connect_to(self, dst_node, *, when: Callable[..., bool]):
         self.add_vertex(Vertex(dst_node, when))
 
+    def debug_cmd(self):
+        cmd = self.tactic.debug_cmd()
+        if isinstance(cmd, dict):
```

Comment:
Why the function needs to handle the conversion?

Reply:
So we can have overloap of `debug_cmd` that do this":
```python
def debug_cmd():
   return Line(...)
```
instead of 
```python
def debug_cmd():
   return [Line(...)]
```